### ISSUE TITLE
Tried to upload to Heroku now having trouble connecting to server

### ISSUE THREAD

**Issue by jcmeyer10** [ga-wdi-boston/independent-work#12]
I will be moving into the independent work room during the next break.  Should be soon.  In the meantime, here is the error I am running in to.  I checked all other terminals, and nothing is running on localhost3000.  

~/WDI/WDI-Projects/tap-app-api (master)$ rails s
=> Booting Puma
=> Rails 4.2.4 application starting in development on http://localhost:3000
=> Run `rails server -h` for more startup options
=> Ctrl-C to shutdown server
A server is already running. Check /Users/JeffMeyer/WDI/WDI-Projects/tap-app-api/tmp/pids/server.pid.
Exiting

**Comment by gaand** [comment]
If no server is running, you may just need to remove the file mentioned.
You can check for a running process by running `ps -p <pid>` where pid is the number in that file.

**Comment by jcmeyer10** [comment]
In the console:
~/WDI/WDI-Projects/tap-app-api (master)$ ps -p <pid>
-bash: syntax error near unexpected token `newline'

But when I followed the file path in the original error, I found this 18750

**Comment by jcmeyer10** [comment]
PID TTY           TIME CMD
18750 ttys009    0:03.75 /usr/local/var/rbenv/versions/2.2.3/bin/ruby bin/rail

**Comment by jcmeyer10** [comment]
Needed to put in number for argument.  So I ran ps -p 18750, got this return 
PID TTY TIME CMD
18750 ttys009 0:03.75 /usr/local/var/rbenv/versions/2.2.3/bin/ruby bin/rail
letting me know there was a process running.

Ran kill -hup 18750 (hangs up)

Then ran ps -p 18750, got this 
  PID TTY           TIME CMD
to confirm there were no processes running

ran rails s to start server and worked like a charm.
